lambdas/export_rds_snapshot_to_s3/main.py

```diff
@@ -13,14 +12,12 @@ def lambda_handler(event, context):
     print("## EVENT")
     print(event)
 
+    snapshot_identifier = event["detail"]["SourceIdentifier"]
+    source_arn = event["detail"]["SourceArn"]
     bucket_name = os.environ["BUCKET_NAME"]
     iam_role_arn = os.environ["IAM_ROLE_ARN"]
     kms_key_id = os.environ["KMS_KEY_ID"]
```

Comment:
Better to move the env variables outside the functions, in order to avoid reading these variables every time the function is invoked